micronaut-camunda-bpm-feature/src/test/kotlin/info/novatec/micronaut/camunda/bpm/feature/test/MnAdminUserConfigurationTest.kt

```diff
@@ -0,0 +1,137 @@
+package info.novatec.micronaut.camunda.bpm.feature.test
+
+import info.novatec.micronaut.camunda.bpm.feature.Configuration
+import javax.inject.Inject
+import info.novatec.micronaut.camunda.bpm.feature.MnAdminUserConfiguration
+import org.camunda.bpm.engine.ProcessEngine
+import io.micronaut.context.ApplicationContext
+import io.micronaut.core.value.PropertyNotFoundException
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import org.camunda.bpm.engine.authorization.Groups
+import org.camunda.bpm.engine.authorization.Authorization
+import org.camunda.bpm.engine.authorization.Resources
+import org.camunda.bpm.engine.identity.User
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import java.util.*
+
+/**
+ * Tests for [info.novatec.micronaut.camunda.bpm.feature.MnAdminUserConfiguration]
+ *
+ * @author Titus Meyer
+ */
+@MicronautTest(rebuildContext = true)
+internal class MnAdminUserConfigurationTest {
+    @Inject
+    lateinit var configuration: Configuration
+
+    @Inject
+    lateinit var mnAdminUserConfiguration: Optional<MnAdminUserConfiguration>
+
+    @Test
+    fun noAdminUserDefined() {
+        Assertions.assertNotNull(configuration)
+        Assertions.assertThrows(PropertyNotFoundException::class.java) { configuration.adminUser.id }
+        Assertions.assertFalse(mnAdminUserConfiguration.isPresent)
```

Comment:
we're testing internal logic here - IMHO the test shouldn't care if the bean is there or not. We should only check if the result is ok, i.e. no user exists.

Quote from https://micronaut-projects.github.io/micronaut-test/latest/guide/
> One of the design goals of Micronaut was to eliminate the artificial separation imposed by traditional frameworks between function and unit tests due to slow startup times and memory consumption.